analyzers/its/sources/SkipGenerated.CS/SkipGenerated.CS.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SkipGeneratedFiles</RootNamespace>
     <AssemblyName>SkipGeneratedFiles</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
For this IT, we don't care what TFM it actually targets. 

So I'm bumping it to the latest to avoid the need to install .NET 4.5.2 to work with this little toy project.